be/src/vec/core/block.cpp

```diff
@@ -64,7 +64,14 @@ Block::Block(const std::vector<SlotDescriptor*>& slots, size_t block_size) {
 }
 
 Block::Block(const PBlock& pblock) {
-    CHECK(HeartbeatServer::check_be_exec_version(pblock.be_exec_version()));
+    int be_exec_version = 0;
+    if (pblock.has_be_exec_version()) {
+        be_exec_version = pblock.be_exec_version();
+    } else {
+        LOG(WARNING) << "pblock.has_be_exec_version()=false, maybe pblock not be serialized well, "
```

Comment:
if old BE do not have the version, the log seems will print everywhere

Reply:
Agreed, remove this log